package.json

```diff
@@ -1,110 +1,110 @@
-{
```

Comment:
Any reason why the whole package.json file seems to be changed?
Seems to have something with this commit: "raise ui-components dependency version to 0.14.0" (7d4c6334497f3d7e898d694d42bcf619fe426ec0)

Reply:
I was wondering the same, though I have no idea why that is.

Reply:
It seems that those are the line endings from Windows (CRLF) which causes this. Can be seen when comparing to `develop`-branch which does not have those.

Reply:
Ah, that would explain it. Actually I think I can get rid of that commit altogether since the version number was updated later as well.

Reply:
Looks better now.